tools/testsys/src/aws_resources.rs

```diff
@@ -49,13 +59,74 @@ impl AwsK8s {
         sonobuoy_mode: SonobuoyMode,
     ) -> Result<Vec<Crd>> {
         let crds = vec![
-            self.eks_crd("", testsys_images)?,
-            self.ec2_crd("", testsys_images)?,
-            self.sonobuoy_crd("", "-test", sonobuoy_mode, None, testsys_images)?,
+            self.eks_crd(testsys_images)?,
+            self.ec2_crd(testsys_images, None)?,
+            self.sonobuoy_crd("-test", sonobuoy_mode, None, testsys_images)?,
         ];
         Ok(crds)
     }
 
+    /// Creates 5 `Test` crds for migration testing. Consisting of a `quick` test, a migration to
+    /// the target version, a `quick` test to validate the migration, a migration to the starting
+    /// version, and a `quick` test to validate the second migration.
```

Comment:
nit: this sort of comment seems like it could get outdated pretty quickly - it's specific in both the number and order of the tests.

It might be better to document each of the steps inline, since there's a natural separation after each `depends_on.push()` to talk about what & why each CRD is added.